runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -678,10 +709,10 @@ public void handleClick(MenuOptionClicked event)
 		}
 
 		if ((event.getWidgetId() == WidgetInfo.BANK_ITEM_CONTAINER.getId()
-				|| event.getWidgetId() == WidgetInfo.BANK_INVENTORY_ITEMS_CONTAINER.getId())
+			|| event.getWidgetId() == WidgetInfo.BANK_INVENTORY_ITEMS_CONTAINER.getId())
```

Comment:
I would avoid making formatting changes to something that is already in there.

Reply:
Thank you. I will do better to catch that.